app/core/services/stt_google_v2.py

```
@@ -0,0 +1,264 @@
+        interim_results: bool = True,
+        single_utterance: bool = False,  # v2 StreamingRecognitionFeatures 미지원 — 무시됨
+        speech_phrases: Optional[List[str]] = None,
+    ) -> AsyncIterator[STTStreamEvent]:
+        """
+        v1 GoogleSTT.stream_events()와 동일한 시그니처.
+        primary_language가 주어지면 language_codes의 첫 번째 자리에 배치.
+        """
+        # primary_language가 들어오면 맨 앞에 두고 나머지는 config 기본값 유지
+        if primary_language and primary_language not in self.config.language_codes:
+            lang_codes = [primary_language] + [
+                l for l in self.config.language_codes if l != primary_language
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)